assets/blocks/jobs/edit.jsx

```diff
@@ -0,0 +1,90 @@
+
+/**
+ * WordPress Dependencies.
+ */
+const { Component } = wp.element,
+	{ InspectorControls } = wp.editor,
+	{ __ } = wp.i18n,
+	{ TextControl, ToggleControl } = wp.components;
+
+/**
+ * Internal Dependencies.
+ */
+import Sidebar from './sidebar.jsx';
```

Comment:
Most style guides have you put imports before const declarations, maybe this isn't the Gutenberg way, but it might be nice to flip this to stay consistent with other es6 best practices out there

Reply:
Done. In Gutenberg itself the WordPress Dependencies are actually `import`'s, but in this case I think it makes more sense to use assignment since they're already available in a global. No need to add extra stuff to our build.

But yes, since it's an assignment instead of an `import`, I agree that it should be under the other `import`'s 🙂